src/policy/protect_payload.rs

```diff
@@ -117,6 +135,25 @@ impl PolicyModule for ProtectPayloadPolicy {
         // Step 3: Unlock memory
         mctx.pmp.set_inactive(POLICY_OFFSET, 0x80400000);
```

Comment:
I a not sure how I feel about hard-coded constants there, keep in mind that we will need to get it to work on the board eventually, so those addresses won't work.

There is a `TARGET_PAYLOAD_ADDRESS` constant, would that work instead?

Reply:
This is not part of the commit, I guess we should fix this in another one